.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,4 +12,5 @@ Point out areas that need special attention or support during the review process
 ### Testing:
 How is this change tested (unit tests, fuzz tests, etc.)? Are there any testing steps to be verified by the reviewer?
 
-By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
+By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and 
+the ISC license.
```

Comment:
Do we want this to be and, or should it be or like the other files you changed

Reply:
Yes, new contributions are to be made under "Apache AND ISC", so that AWS-LC can used under either "Apache2.0 OR ISC".